### ISSUE TITLE
canView permissions aren’t checked on front-end

### ISSUE THREAD

**Issue by kinglozzer** [dnadesign/silverstripe-elemental#253]
I’ve implemented `canView()` permissions on a per-element basis (mimicking `CanViewType` and `ViewerGroups` from `SiteTree`), out of the box these permissions aren’t checked before rendering.

It’s easy enough to implement these checks myself, just struck me as worth considering :)

**Comment by robbieaverill** [comment]
Fancy a pull request @kinglozzer? :)

**Comment by raissanorth** [comment]
Replicated the described issues using:

```
<?php

use DNADesign\Elemental\Models\ElementContent;

class MySweetElement extends ElementContent
{

    function canView($member = null) {
        die("called");
        return false;
    }

    public function getType()
    {
        return ('MySweetElement');
    }
}
```

**Comment by robbieaverill** [comment]
Fixed with #259
